docs/UserGuide.md

```diff
@@ -247,20 +248,20 @@ Examples:
 *  `sort appointment o/asc k/datetime` Sorts `appointment` by `datetime` in ascending order.
 *  `sort property o/desc k/price` Sorts `property` by `price` in descending order.
 
-### Searching
+### 3.8 Searching
 
-#### Searching properties: `find property`
+#### 3.8.1 Searching properties: `find property`
 
 Finds properties that match the criterion provided.
 
 Formats:
-* `find property [KEYWORD] [OPTION...]`
+* `find property [KEYWORD]... [OPTION]...`
 
 Description:
 * There can be 0 or more keywords and 0 or more options, but keywords and options cannot be both empty. All text are case insensitive. 
 
 Options:
-* `t/[PROPERTY_TYPE]`
+* `[t/PROPERTY_TYPE]`
 
     Search for properties whose housing type field contain patterns specified in `[REMARKS]`. 
```

Comment:
is this supposed to be `[PROPERTY_TYPE]` instead?

Reply:
The format is supposed to be `t/hdb` for example

Reply:
I meant line 265 266, there's this `[remarks]`, this that supposed to be the case?

Reply:
ohh, thanks for catching this